Documentation/Blazorise.Docs/Pages/Docs/Components/Dates/DateEditPage.razor

```diff
@@ -64,6 +64,25 @@
     <DocsPageSectionSource Code="DateEditDateTimeExample" />
 </DocsPageSection>
 
+<DocsPageSubtitle>
+    Functions
```

Comment:
I think we've used the name `Methods` on other places, `Datagrid`, `Autocomplete`, I honestly have no real preference. But I think in C# the naming `Method` is used more?  We should choose one or the other.

Reply:
Agree. Methods make more sense.